### DISCUSSION TITLE
Foreground service support?

### DISCUSSION THREAD

**Discussion by Sebosek** [shinyorg/shiny#1043] [Q&A] [answered]
Hi,

not sure, if I've misunderstood something, but Shine doesn't support foreground service, does it?
In docs I've found only a job, which looks like a periodical scheduler, do I get it right?
Thanks.

S.

**Comment by aritchie** [comment, marked as answer]
Shiny uses android foreground services for GPS (Shiny.Locations) & Beacon Monitoring (Shiny.Beacons).  

Jobs are periodic and do not use foreground services.  I only use foreground services to get equivalent behavior with other platforms.
